server/middleware.go

```diff
@@ -3,6 +3,7 @@ package server
 import (
 	"bytes"
 	"encoding/json"
+	"golang.org/x/net/context"
```

Comment:
Would you mind putting this import in its own block, separate from the stdlib packages?

https://github.com/golang/go/wiki/CodeReviewComments#imports